object/interp.go

```diff
@@ -0,0 +1,35 @@
+package object
+
+import (
+	"errors"
+)
+
+var commands map[string]Extension
+
+func Init() {
+	commands = make(map[string]Extension)
+}
+
+func CreateCommand(cmd Extension) error {
+	if commands == nil {
+		Init()
+	}
```

Comment:
What about this?

I mean fonuou need an Init?

```suggestion
func CreateCommand(cmd Extension) error {
	if commands == nil {
		  commands = make(map[string]Extension)
	}
```

Reply:
I keep Init() so I and others can reset the list of functions (I haven't done it yet in object_test but I do in token_test)

Reply:
added godoc and better names in latest commit